<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>editpage</class>
 <widget class="QDialog" name="editpage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>465</width>
    <height>327</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog
{
      background: qconicalgradient(cx:0.5, cy:0.5, angle:30,
                stop:0 rgb(193, 192, 193), stop:1 rgb(255, 159, 159))

}
QLineEdit
{
 border-width: 1px;
    border-style: solid;
    border-color:rgb(162, 10, 23);
    border-radius: 4px;
background-color:white;
color:rgb(166, 5, 0);
	font: 13pt &quot;Comic Sans MS&quot;;
}
QLabel
{
	
	font: 75 17pt &quot;Comic Sans MS&quot;;
color:black;
}
QPushButton
{
	
	font: 75 14pt &quot;Comic Sans MS&quot;;
background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
                stop:0 white, stop: 0.4 white, stop:1 rgb(193, 192, 193));
color:black;
  	 border-width: 1px;
    border-style: solid;
    border-radius: 4px;
}</string>
  </property>
  <widget class="QPushButton" name="editbutton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>280</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>edit</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>341</width>
     <height>231</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="editname">
        <property name="text">
         <string>name :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="editconsumer">
        <property name="text">
         <string>consumer :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="edittype">
        <property name="text">
         <string>type :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="editnumber">
        <property name="text">
         <string>number :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="editprice">
        <property name="text">
         <string>price :</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLineEdit" name="name"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditconsmer"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdittype"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditnumber">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditprice">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
